ko/admin/config.rst

```diff
@@ -2351,6 +2351,43 @@ cubrid_broker.conf 설정 파일과 기본 제공 파라미터
 |                                 |                         | SQL_LOG                         | string | ON                           | 가능      |
 |                                 |                         +---------------------------------+--------+------------------------------+-----------+
 |                                 |                         | SQL_LOG_MAX_SIZE                | KB     | 10000                        | 가능      |
+|                                 | 샤드                    | SHARD                           | string | OFF                          |           |
```

Comment:
There is no dividing line between logs and shards.

Reply:
I'll add it right away